core/prelude-global-keybindings.el

```diff
@@ -46,7 +46,8 @@
 (global-set-key (kbd "C-^") 'crux-top-join-line)
 
 ;; Start proced in a similar manner to dired
-(unless (eq system-type 'darwin)
+(unless (or (eq system-type 'darwin)
```

Comment:
I guess we can just remove this `unless`. I assume very few people are using older Emacsen at this point.

Reply:
Sorry for the delay. I've updated the patch according to your suggestion.